cluster/validate-cluster.sh

```diff
@@ -44,7 +44,7 @@ while true; do
   found=$(echo "${nodes_status}" | tr "," "\n" | grep -c 'Ready:') || true
   ready=$(echo "${nodes_status}" | tr "," "\n" | grep -c 'Ready:True') || true
 
-  if (( "${found}" == "${EXPECTED_NUM_NODES}" )) && (( "${ready}" == "${EXPECTED_NUM_NODES}")); then
+  if (( "${found}" >= "${EXPECTED_NUM_NODES}" )) && (( "${ready}" >= "${EXPECTED_NUM_NODES}")); then
```

Comment:
Please you add a "warning" output that the number of nodes was higher than expected. It may help in debugging in case this mismatch becomes an issue.